### PR TITLE
Refactor: Enrollment index

### LINKED ISSUE #1479: Enrollment Index - Remove media item (All flows)
## Desktop

Image

## Mobile

Image

### LINKED ISSUE #1480: Enrollment Index - Update copy (All flows)
- [x] ALL flows - **COPY** change title to `Your eligibility is confirmed! You’re almost there.`
- [x] ALL flows - **COPY ONLY** payment media item - change copy on Littlepay media item

### PR DESCRIPTION
**Pull request by thekaveman**
Closes #1479 
Closes #1480 (copy)

Does not change how the sign out button works - this was getting a little more complicated and I didn't want to block this PR any longer. Plus that may depend on what happens in #1508.

### CODE REVIEW

File: benefits/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -602,26 +602,24 @@ msgstr ""
...
-"Por favor utilice una tarjeta de crédito o débito Visa o Mastercard sin "
-"contacto emitida por un banco."
+msgid "enrollment.pages.index.link_card_item.p[0]s[0]"
+msgstr "Será dirigido a nuestro socio de pago, "
+
+msgid "enrollment.pages.index.link_card_item.p[0]s[1]"
+msgstr "No almacenamos su información y no se le cobrará."
+
+msgid "enrollment.pages.index.link_card_item.p[1]s[0]"
+msgstr "Utilice una tarjeta de débito o crédito de Visa o Mastercard.."
```
Comment (machikoyasuda) [inline benefits/locale/es/LC_MESSAGES/django.po:616]: @thekaveman Directly added new Spanish copy into this PR!

Comment (machikoyasuda) [reply benefits/locale/es/LC_MESSAGES/django.po:616]: image

File: benefits/enrollment/templates/enrollment/includes/media_item__bankcardcheck__index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    {% include "core/includes/icon__direct_args.html" with name="bankcardcheck" alt=icon_alt %}
+{% endblock icon %}
+
+{% block heading %}
+    {% translate "enrollment.pages.index.link_card_item.heading" %}
+{% endblock heading %}
+
+{% block body %}
+    <div class="media-body--details">
+        {% url "core:help" as help_page %}
```
Comment (machikoyasuda) [inline benefits/enrollment/templates/enrollment/includes/media_item__bankcardcheck__index.html:15]: Is this line still necessary?

Comment (thekaveman) [reply benefits/enrollment/templates/enrollment/includes/media_item__bankcardcheck__index.html]: Rebased and fixed

**Comment by machikoyasuda** [comment]
🔥🔥🔥
image

File: benefits/enrollment/templates/enrollment/index.html [hunk 3]
```diff
@@ -1,108 +1,113 @@
 {% extends "core/base.html" %}
+{% load i18n %}
+
+{% translate "core.buttons.wait" as loading_text %}
+
+{% block page_title %}
+  {% translate "enrollment.pages.index.title" %}
```
Comment (machikoyasuda) [inline benefits/enrollment/templates/enrollment/index.html:7]: Does this line need `&nbsp;|&nbsp;`?

Comment (machikoyasuda) [reply benefits/enrollment/templates/enrollment/index.html:7]: image

File: benefits/locale/es/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -596,35 +581,37 @@ msgctxt "image alt text"
...
-"\">Littlepay</a>. No almacenamos su información y no se te cobrará."
+msgid "enrollment.pages.index.link_card_item.p[0]s[1]"
+msgstr "No almacenamos su información y no se le cobrará."
 
-msgid "enrollment.pages.index.link_card_item.p[1]"
-msgstr ""
-"Por favor utilice una tarjeta de crédito o débito Visa o Mastercard sin "
-"contacto emitida por un banco."
+msgid "enrollment.pages.index.link_card_item.p[1]s[0]"
+msgstr "Utilice una tarjeta de débito o crédito de Visa o Mastercard.."
```
Comment (machikoyasuda) [inline benefits/locale/es/LC_MESSAGES/django.po:596]:
```suggestion
msgstr "Utilice una tarjeta de débito o crédito de Visa o Mastercard."
```

File: benefits/enrollment/templates/enrollment/index.html (hunk 3 above)
Comment (thekaveman) [reply benefits/enrollment/templates/enrollment/index.html]: Fixed and rebased

**Comment by machikoyasuda** [comment]
This is not something I think we should change in this PR or any of the subsequent copy PRs, but I wanted to note this:
image

The Media Item parent container width looks a bit too wide now. Not sure if this is a change in Figma, copy or what. It's something we can look into at the very very very end with @srhhnry once we finish everything else first, I think.

**Review by machikoyasuda — APPROVED** [review]

File: benefits/enrollment/templates/enrollment/index.html [hunk 5]
```diff
@@ -1,108 +1,113 @@
...
+    <ul class="media-list mx-0 px-0 d-flex justify-content-center flex-column">
+      {% include "enrollment/includes/media-item--bankcardcheck--index.html" %}
+    </ul>
+  </div>
+  {% comment %}
+    This Javascript code is partially generated by this template and so it must
+    come before the forms, which are rendered at just before the {% endblock inner-content %}
+  {% endcomment %}
+  {% url "enrollment:token" as access_token_url %}
+  {% translate "core.buttons.wait" as loading_text %}
```
Comment (angela-tran) [inline benefits/enrollment/templates/enrollment/index.html:31]: I noticed we have this code on line 4 and this line 31. Is there a reason we need both?

File: benefits/enrollment/templates/enrollment/index.html [hunk 6]
```diff
@@ -1,108 +1,113 @@
...
+  {% for f in forms %}
     {% include "core/includes/form.html" with form=f %}
   {% endfor %}
 {% endblock inner-content %}
 
 {% block call-to-action-button %}
-  {% if payment_processor %}
-    <button type="button" class="btn btn-primary btn-lg loading" role="status" disabled>
-      {{ payment_processor.loading_text }}
+  <button type="button" class="btn btn-primary btn-lg loading" role="status" disabled>{{ loading_text }}</button>
```
Comment (angela-tran) [inline benefits/enrollment/templates/enrollment/index.html:107]: I'm seeing something weird where this `loading_text` doesn't show up. The one on line 46 works though 🤔 

benefits-enrollment-index

**Review by angela-tran — COMMENTED** [review]
These code changes look good to me. I'm seeing something weird with the `loading_text` though. Not sure if it's just me or what might be happening there.

**Comment by thekaveman** [comment]
Thanks for that find @angela-tran! I think it is because the variables aren't available outside of the `{% block %}` tags.

In [3f8017e](https://github.com/cal-itp/benefits/pull/1502/commits/3f8017ed4bea6cdd8f5c08b7a09d740134650dea) I moved the extra translate to within the button block.

File: benefits/enrollment/templates/enrollment/index.html (hunk 5 above)
Comment (thekaveman) [reply benefits/enrollment/templates/enrollment/index.html:29]: It turns out we do need both, but the other one was on the wrong line: https://github.com/cal-itp/benefits/pull/1502#issuecomment-1629832107

File: benefits/enrollment/templates/enrollment/index.html (hunk 6 above)
Comment (machikoyasuda) [reply benefits/enrollment/templates/enrollment/index.html]: yeah I noticed this on mine too.

**Review by angela-tran — APPROVED** [review]
🙌

**Pull request merged**